Symfony/CS/Fixer/Contrib/LongArraySyntaxFixer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Gregor Harlan <gharlan@web.de>
+ */
+class LongArraySyntaxFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        foreach ($tokens as $index => $token) {
+            if (!$tokens->isShortArray($index)) {
+                continue;
+            }
+
+            $closeIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_SQUARE_BRACE, $index);
+
+            $tokens[$index]->setContent('(');
+            $tokens[$closeIndex]->setContent(')');
+            $tokens->insertAt($index, new Token(array(T_ARRAY, 'array')));
```

Comment:
You are inserting new Token inside collection that you are currently iterating via foreach.

Take a look for a simple example:
```php
<?php

$foo = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
$visited = 0;

foreach ($foo as $k => $v) {
    ++$visited;

    if ($k % 5 === 0) {
        $foo[] = "$k = $v";
    }

    // handle $k, $v here
}

var_dump(count($foo));
var_dump($visited);
```

Here, we are iterating thru every element, sometimes add a new element to collection.
After iteration collection size is 12, but... we visited only 10 elements. 
That means we won't handle some elements of collection.

Same case is in your code.

Please iterate backward, like:
```php
for ($index = $tokens->count() - 1; 0 <= $index; --$index) { /*...*/ }
```